src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml

```diff
@@ -9,16 +9,40 @@
 
     <!--  #Design  -->
     <StackPanel Margin="5,0">
+        <Grid x:Name="ActivateMoreGrid" Margin="0,0,0,5">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
```

Comment:
Give a name to your 2 buttons  and 2 textblock and define the 2 buttons in the same column/row (same thing for the textblock) and use the visibility from the code behind (much more readable IMO).